src-self-hosted/translate_c.zig

```diff
@@ -5073,6 +5080,39 @@ fn parseCExpr(c: *Context, it: *CTokenList.Iterator, source: []const u8, source_
             if_node.@"else".?.body = try parseCPrimaryExpr(c, it, source, source_loc, scope);
             return &if_node.base;
         },
+        .Comma => {
+            _ = try appendToken(c, .Semicolon, ";");
+            const block_scope = try Scope.Block.init(c, scope, "blk");
```

Comment:
Do we need some sort of uniqueness check on `blk`?

Reply:
Not currently at least.
```zig
test "" {
    var a: u8 = blk: {
        var b: u8 = blk: {
            break :blk 2;
        };
        break :blk b + 2;
    };
    @import("std").debug.assert(a == 4);
}
```

Reply:
Awesome! perhaps add a test so we don't accidentally break it in future?

Reply:
It's fine to leave this code, but just so you know it is exploiting a missing compile error, so it will become a compile error in the future. #665